ex1/main.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import "fmt"
+
+
+
+func main() {
+
+
+var OS int
```

Comment:
Why are your internal variables still having capitalised first letters, make them small letters, they are not exported.